main.cpp

```diff
@@ -514,23 +585,40 @@ void Airport::openRestaurant(int PriceOfRestaurant, int spaceNeededForRestaurant
 {
     if ((PriceOfRestaurant <= 300000) && (spaceNeededForRestaurant <= 3000))
     {
-        std::cout << "Buy restaurant" << std::endl;
+        std::cout << "\nBuy the space for restaurant." << std::endl;
+    }
+    else
+    {
+        std::cout << "\nDo not buy space for restaurant." << std::endl;
     }
 }
 
+Airport::Airport()
+{
+    std::cout << "Airport being constructed!" << std::endl;
+}
+
 void Airport::passengerReachDestination(std::string passengerTicketDestination, std::string 
     planeLandedAtDestination)
```

Comment:
Don't split expressions like this onto multiple lines.
split at the `,`:
```
void function(int a, int b,
              int c, int d)
{
    statements();
}
```
- [ ] revise